src/components/GroceryItem/GroceryItem.js

```diff
@@ -47,16 +48,49 @@ const GroceryItem = ({ item }) => {
       });
   };
 
-  const resetDate = () => {
+  // Should we delete the option to uncheck? Otherwise, we can't really reset the previousPurchaseDate in Firestore.
+  const handleUncheckBox = () => {
     setChecked(false);
   };
 
+  const getAriaLabel = () => {
+    if (
+      currentDate - item.purchaseDate >=
+        2 * (item.nextPurchaseDate - item.purchaseDate) ||
+      item.numberOfPurchases === 0
+    ) {
+      return 'Inactive';
+    } else if (item.howSoon < 7) {
+      return 'Need to buy soon';
+    } else if (item.howSoon >= 7 && item.howSoon <= 30) {
+      return 'Need to buy kind of soon';
+    } else if (item.howSoon > 30) {
+      return 'Do not need to buy soon';
```

Comment:
Do you all think we should lower 30 down to say 21? Otherwise, items will rarely be green?

Reply:
Lowering down to 21 sounds a great idea for the reason you mentioned!

Unrelated thought: As you mention changing that value (30 to 21).... I think this could be a good opportunity to define some constants with those timeframes to define the frequency.

e.g.
```js
const HIGHEST_PERIOD= 7;
const LOWEST_PERIOD = 30 (or 21);
...
} else if (item.howSoon < HIGHEST_PERIOD) {
  return 'Need to buy soon';
} else if (item.howSoon >= HIGHEST_PERIOD && item.howSoon <= LOWEST_PERIOD) {
...
```

No need to make any changes tbh, but wanted to see if you find this useful.